service/resource_router_test.go

```diff
@@ -0,0 +1,90 @@
+package service
```

Comment:
For me those kind of tests are cluttering the code. This check if golang map implementation is correct IMO.

Reply:
Do you prefer to remove it? I've used to test the empty string to be sure

Reply:
Yes, I would remove them.

Reply:
Why removing them? Using maps is an implementation detail the tests do not care about. Making sure the implementation works is the purpose of the tests. I would like to keep them. They might be improvable, but then we should rather improve them instead of removing them.